src/web/topic/components/TopicWorkspace/MoreActionsDrawer.tsx

```diff
@@ -150,6 +130,46 @@ export const MoreActionsDrawer = ({
   const expandDetailsTabs = useExpandDetailsTabs();
 
   const [resetDialogOpen, setResetDialogOpen] = useState(false);
+  const [screenshotDialogOpen, setScreenshotDialogOpen] = useState(false);
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<ScreenshotForm>({
+    defaultValues: defaultResolution,
+    resolver: zodResolver(ScreenshotSchema),
+  });
```

Comment:
```suggestion
  } = useForm<ScreenshotForm>({
    mode: "onBlur",
    reValidateMode: "onBlur",
    defaultValues: defaultResolution,
    resolver: zodResolver(ScreenshotSchema),
  });
```
I think validation on blur feels a little nicer (the default is to wait until first submit, then validate on on change after that)